interface/billing/billing_report.php

```diff
@@ -348,7 +348,6 @@ function toencounter(pid, pubpid, pname, enc, datestr, dobstr) {
             encurl = 'patient_file/encounter/encounter_top.php?set_encounter=' + encodeURIComponent(enc) +
                 '&pid=' + encodeURIComponent(pid);
             paturl = 'patient_file/summary/demographics.php?pid=' + encodeURIComponent(pid);
-            parent.clearPatient();
```

Comment:
This also ensures replaced patient info is removed before new patient is activated. This includes session data.
All these functions need to be made into an async/await pattern because if you note that depending on the environment this pattern is ran under different results can happen. 
For the short term perhaps we create a couple new functions with es6 awaits to cover this requirement.

Reply:
Looks like biggest problem is that even if demographic tab is called first because it requires more time than encounter to load when its tab is set then when demo load finishes it switches back to its tab. I created a promised based loadframe await to test to no avail! Need to check how demo scripts returns the ajax call if any to resolve the promise....
have headache, maybe tomorrow...